src/main/java/uk/gov/hmcts/reform/migration/service/DataMigrationServiceImpl.java

```diff
@@ -37,7 +37,8 @@ public class DataMigrationServiceImpl implements DataMigrationService<Map<String
     private final ObjectMapper objectMapper;
     private final Map<String, Function<Map<String, Object>, Map<String, Object>>> migrations = Map.of(
         "DFPL-log", this::triggerOnlyMigration,
-        "DFPL-2527", this::triggerOnlyMigration
+        "DFPL-2527", this::triggerOnlyMigration,
```

Comment:
Is there a reason you left this in?

Reply:
I have removed scripts that already done.